src/commands.rs

```diff
@@ -1575,19 +1575,23 @@ struct BackoutArgs {
 #[derive(clap::Args, Clone, Debug)]
 struct BranchArgs {
     /// The branch's target revision
-    #[clap(long, short, default_value = "@")]
+    #[clap(long, short, default_value = "@", group = "action")]
```

Comment:
Oh, I didn't know you could do it this way. I did it differently for `RebaseArgs` a bit further up (line 1541). Maybe your way is preferred in simple cases like this (the`ArgGroup` way is probably more flexible, but typically we just want a simple group like here).